xml/System.Linq/Enumerable.xml

```diff
@@ -4412,7 +4490,7 @@ Only unique elements are returned.
   
  <xref:System.Linq.Enumerable.GroupJoin%2A> has no direct equivalent in traditional relational database terms. However, this method does implement a superset of inner joins and left outer joins. Both of these operations can be written in terms of a grouped join. See [Join Operations](https://msdn.microsoft.com/library/442d176d-028c-4beb-8d22-407d4ef89107).  
   
- In query expression syntax, a `join … into` (Visual C#) or `Group Join` (Visual Basic) clause translates to an invocation of <xref:System.Linq.Enumerable.GroupJoin%2A>.  
+ In query expression syntax, a `join � into` (Visual C#) or `Group Join` (Visual Basic) clause translates to an invocation of <xref:System.Linq.Enumerable.GroupJoin%2A>.  
```

Comment:
Is this intentional? Looks like a replacement character is being used here.

Reply:
It's an encoding issue -- it shows up right in an editor that understands codepage 1252. They're checking which it should be. I'll not submit until it's figured out.

Reply:
For now, we can revert these in the PR as suggestions (actually, I want to just use three points instead of the non-ascii ellipsis character):

```suggestion
 In query expression syntax, a `join ... into` (Visual C#) or `Group Join` (Visual Basic) clause translates to an invocation of <xref:System.Linq.Enumerable.GroupJoin%2A>.  
```